src/internal/connector/onedrive/drive.go

```diff
@@ -170,6 +170,9 @@ func defaultItemPager(
 		link,
 		[]string{
 			"content.downloadUrl",
+			// https://learn.microsoft.com/en-us/onedrive/developer/rest-api
+			// /concepts/scan-guidance#scanning-permissions-hierarchies
+			"oneDrive.sharedChanged",
```

Comment:
Does this cover sharepoint as well?

Reply:
Good questions. I did not check it for SharePoint as we still don't have permissions for SharePoint. This was added for `@microsoft.graph.sharedChanged`, but we are not actively using it as of now.

Reply:
I'm removing it for now, will add it to the TODO to be picked up later.